[master < T921] Add documentation for NULLIF command in LOAD CSV (#921)
memgraph/docs: cypher-manual/clauses/load-csv.md

* `QUOTE <quote-string>` option enables you to specify the CSV quote character.
  If it isn't set, the default quote character `"` is assumed.

> * `NULLIF <nullif-string>` option enables you to specify the character sequence which
>   will be parsed as null.
>   If it isn't set, no character will be treated as null. By default, all empty columns
>   in Memgraph are treated as empty strings.

Reviewer: ```suggestion
* `NULLIF <nullif-string>` option enables you to specify a sequence of characters that will be parsed as null.
  By default, all empty columns in Memgraph are treated as empty strings, so if this option is not used, no values will be treated as null. 
```
